hero_python_port/hero_py/survival.py

```
@@ -0,0 +1,156 @@
+# hero_python_port/hero_py/survival.py
+import numpy as np
+from typing import Protocol, runtime_checkable
+from scipy.stats import exponweib, weibull_min, lognorm, loglaplace # Using scipy.stats for implementations
```

Ruff (0.11.9)

`scipy.stats.exponweib` imported but unused

Remove unused import: `scipy.stats.exponweib`

(F401)